pkg/config/config.go

```diff
@@ -0,0 +1,33 @@
+package config
+
+import (
+	"crypto/tls"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+type Config struct {
+	Addr    string `env:"GRPC_ADDR" envDefault:"grpc.constantine.archway.tech:443"`
```

Comment:
Should this be localhost and not constantine, reason when you are developing/testing you might want to use local instead of constantine?

Reply:
It is `constantine` by default to allow to play with the exporter as easy as possible. Doing local setup and having working grpc endpoint is an extra effort.